submissions/Lu-sure/Popup/index.html

```diff
@@ -17,8 +17,8 @@
             <a href="#" class="nav-text-item">Images</a>
           </li>
 
-          <li class="nav-img-item menu-item-border">
-            <input type="checkbox" id="hoogle-menu" class="hide-checkbox">
+          <li class="nav-img-item" tabindex="0">
```

Comment:
This approach makes unaccessible your popup from the keyboard(Try to open your popup without using mouse). Your HTML structure looks good and if you remove all `tabindex` attributes it will work as expected.

Reply:
I found my mistake. I fixed this and removed all `tabindex` attributes.